gitgud/levels/rampup/_relrefs1/instructions.txt

```diff
@@ -0,0 +1,15 @@
+Now that we've learned how to detach the HEAD, let's learn easier ways to refer to a commit.
+>>>
+In the previous exercise, we learned that we only need 4-digits of a commit hash in order to reference a particular commit. This method made the process much easier than entering in a giant string of characters and numbers (e.g. fed2da64c0efc5293610bdd892f82a58e8cbc5d8). However, in this exercise we'll learn even more ways of refering to commits!
+>>>
+In particular there are two ways of referencing commits called "relative refs." Relative refs are used to refer to a commit nodes that are a couple commits behind your current commit. It should be noted that using relative refs are only efficient to use if the commit you are referring is nearby in the tree; otherwise, in many cases it may be faster to bring up the commit log and use the hash.
+>>>
+The ^ operator refers to the parent node of the commit you specify. I.e., "git checkout bugFix^" will checkout the parent node of bugFix! "^" is shorthand for "^1" meaning parent 1. To further illustrate this point, say commit "C" is the merge node of "A" and "B." "C^2" refers to parent two of C, which is B. For an octopus merge, "^3" would refer to parent 3, and so on. The second operator is the "~<number>" operator. It's used to point to a commit that is several nodes above the node you specify. That operator will be covered in the next exercise. For now, let's take a look at the caret operator.
```

Comment:
`is several nodes above the node you specify` -> `is several commits prior to the node you node you specify`